packages/gcf-utils/test/gcf-bootstrapper.ts

```diff
@@ -271,6 +271,62 @@ describe('GCFBootstrapper', () => {
       downloaded.done();
     });
 
+    it('does not retry the task, if tmpUrl in payload cannot be found (expired)', async () => {
+      await mockBootstrapper();
+      process.env.WEBHOOK_TMP = '/tmp/foo';
+      req.body = {
+        tmpUrl: '/bucket/foo',
+        installation: {id: 1},
+      };
+      req.headers = {};
+      req.headers['x-github-event'] = 'issues';
+      req.headers['x-github-delivery'] = '123';
+      // populated once this job has been executed by cloud tasks:
+      req.headers['x-cloudtasks-taskname'] = 'my-task';
+      // Fake download from Cloud Storage, again with the goal of ensuring
+      // we're using the streams API appropriately:
+      const downloaded = nock('https://storage.googleapis.com')
+        .get('/storage/v1/b/tmp/foo/o/%2Fbucket%2Ffoo?alt=media')
+        .reply(404);
+
+      await handler(req, response);
+
+      delete process.env.WEBHOOK_TMP;
+      sinon.assert.calledOnce(configStub);
+      sinon.assert.notCalled(sendStatusStub);
+      sinon.assert.calledOnceWithMatch(sendStub, {statusCode: 200});
+      sinon.assert.notCalled(spy);
```

Comment:
Nit: naming suggestion: `handerStub`?
Not a blocker.

Reply:
These were the existing stub names, we can rename them in a follow-up.